app/components/c-attribute-grid.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <div :class="b(modifiers)">
+    <div ref="table" :class="b('table', { open: isOpen })" :style="{ 'max-height': maxHeight }">
+      <c-attribute-grid-items :items="attributes"/>
+    </div>
+    <div :class="b('toggle', { open: isOpen })" role="button" @click="toggle()">
+      <e-icon :class="b('arrow')" inline icon="i-arrow-down"/>
+    </div>
+  </div>
+</template>
+
+<script>
+  import cAttributeGridItems from '@/components/c-attribute-grid-items';
+
+  export default {
+    name: 'c-attribute-grid',
+    components: {
+      cAttributeGridItems,
+    },
+    // mixins: [],
+
+    props: {
+      /**
+       * Defines the items displayed in the attribute grid.
+       */
+      attributes: {
+        type: Array,
+        required: true,
+      },
+
+      /**
+       * Defines whether the grid should shrink on mobile or not.
+       */
+      shrinkOnMobile: {
+        type: Boolean,
+        default: false,
+      },
+    },
+
+    data() {
+      return {
+        isOpen: true,
+        openHeight: 0,
+        closedHeight: 0,
+      };
+    },
+
+    computed: {
+      modifiers() {
+        return {
+          shrinkOnMobile: this.shrinkOnMobile,
+          dontShrinkOnMobile: !this.shrinkOnMobile
```

Comment:
Isn't it enough to just have 1 modifier `shrinkOnMobile` because the other is just the opposite?

Reply:
Sure, didn't see that